config/environments/development.rb

```diff
@@ -53,4 +53,10 @@
     config.web_console.whitelisted_ips = '0.0.0.0/0' # all ip-addresses
     config.logger = ActiveSupport::Logger.new(STDOUT) 
   end
+
+  #config.after_initialize do
```

Comment:
Could you also add

    # TODO: Has an issue with no support mongoid v6.0.0, re-asses later

Here too?